### PR TITLE
remove resolve_compose plugin if yum_repourls is used

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -697,7 +697,7 @@ def render_resolve_composes(self):
             return
 
         odcs_url = self.spec.odcs_url.value
-        if not odcs_url:
+        if not odcs_url or self.spec.yum_repourls.value:
```
Comment (vrutkovs) [inline osbs/build/build_request.py]: Lets have a log message about why the plugin was removed

File: tests/build_/test_build_request.py [hunk 2]
```diff
@@ -2791,20 +2791,26 @@ def test_render_resolve_composes(self, odcs_insecure, additional_params, odcs_op
         plugin_args = plugin_value_get(plugins, plugin_type, plugin_name, 'args')
         assert plugin_args == expected_plugin_args
 
-
-    @pytest.mark.parametrize('enabled', (True, False))
-    def test_remove_resolve_composes(self, enabled):
+    @pytest.mark.parametrize(('odcs_url', 'yum_repo', 'raised'), [
```
Comment (vrutkovs) [inline tests/build_/test_build_request.py]: Nitpick - `yum_repos` would be a better name

Comment (rcerven) [reply tests/build_/test_build_request.py]: ok

File: osbs/build/build_request.py (hunk 1 above)
Comment (rcerven) [reply osbs/build/build_request.py]: ok

**Comment by rcerven** [comment]
rn updated:

resolve_compose plugin is removed if yum_repourls is used

**Pull request merged**